osf/external/gravy_valet/translations.py

```diff
@@ -65,26 +63,21 @@ def make_ephemeral_user_settings(gv_account_data, requesting_user):
 
 
 def make_ephemeral_node_settings(gv_addon_data: gv_requests.JSONAPIResultEntry, requested_resource, requesting_user):
-    if gv_addon_data.resource_type == 'configured-citation-addons':
-        include_path = ('base_account', 'external_citation_service')
-        service_key = gv_addon_data.get_included_attribute(
-            include_path=include_path,
-            attribute_name='external_service_name'
-        )
-    else:
-        include_path = ('base_account', 'external_storage_service')
-        service_key = gv_addon_data.get_included_attribute(
-            include_path=include_path,
-            attribute_name='wb_key'
-        )
-    legacy_config = _LegacyConfigsForWBKey[service_key].value
-    return EphemeralNodeSettings(
+    addon_type = gv_addon_data.resource_type.split('-')[1]
+    include_path = ('base_account', f'external_{addon_type}_service')
+    service_wb_key = gv_addon_data.get_included_attribute(
+        include_path=include_path,
+        attribute_name='external_service_name' if addon_type == AddonType.CITATION else 'wb_key'
```

Comment:
This is to consistent with current stage configuration. If `wb_key` for citation addons is configured properly, we may replace it with
```suggestion
        attribute_name='wb_key'
```